src/graphql/types/scalar/display-currency.ts

```diff
@@ -0,0 +1,28 @@
+import { GT } from "@graphql/index"
+import { InputValidationError } from "@graphql/error"
+
+const DisplayCurrency = GT.Scalar<string | InputValidationError>({
+  name: "DisplayCurrency",
+  description: "Display currency of an account",
+  parseValue(value) {
+    if (typeof value !== "string") {
+      return new InputValidationError({ message: "Invalid type for DisplayCurrency" })
+    }
+    return validDisplayCurrencyValue(value)
+  },
+  parseLiteral(ast) {
+    if (ast.kind === GT.Kind.STRING) {
+      return validDisplayCurrencyValue(ast.value)
+    }
+    return new InputValidationError({ message: "Invalid type for Currency" })
+  },
+})
+
+function validDisplayCurrencyValue(value: string): string | InputValidationError {
+  if (!value || value.length < 3) {
```

Comment:
can it be more than 3 ? or 4? maybe having an upper bound could be nice as well

Reply:
fixed https://github.com/GaloyMoney/galoy/pull/2042/commits/7e85659b7c5dcc040dfcf3f026d56472308e192b